simulator/src/main/java/com/hazelcast/simulator/provisioner/Provisioner.java

```diff
@@ -82,28 +82,36 @@ void installAgent(String ip) {
 
         // first we remove the old lib files to prevent different versions of the same JAR to bite us
         bash.sshQuiet(ip, format("rm -fr hazelcast-simulator-%s/lib", getSimulatorVersion()));
-        bash.uploadToAgentSimulatorDir(ip, SIMULATOR_HOME + "/bin/", "bin");
-        bash.uploadToAgentSimulatorDir(ip, SIMULATOR_HOME + "/conf/", "conf");
-        bash.uploadToAgentSimulatorDir(ip, SIMULATOR_HOME + "/jdk-install/", "jdk-install");
+        bash.sshQuiet(ip, format("mkdir -p hazelcast-simulator-%s/lib/", getSimulatorVersion()));
```

Comment:
I made this change because we should re-create the directory first. Otherwise a re-ordering of the lib upload may lead to a (funny) error: If there is just a single JAR file uploaded, it was copied in the file `lib` and no folder was created.